Makefile

```diff
@@ -9,7 +9,12 @@ DOCKER_COMPOSE_CMD = docker-compose -f docker-compose.yml
 ifndef JENKINS_URL
   DOCKER_COMPOSE_CMD += -f docker-compose.development.yml
 endif
-TEST_CMD = $(DOCKER_COMPOSE_CMD) run publishing-e2e-tests bundle exec rspec
+
+ifndef TEST_ARGS
+  TEST_ARGS = --tag ~flaky
+endif
```

Comment:
The conditional isn't necessary here, as the assignment would be ignored if the variable is overridden [1]. However, I don't mind keeping it.

1: https://www.gnu.org/software/make/manual/make.html#Overriding

Reply:
When testing we found it wouldn't pick up variables that weren't explicitly passed to the make command even if they had been 'exported'

e.g.
```bash
make test TEST_ARGS=spec/fun/fun.rb
```
worked
```bash
export TEST_ARGS=spec/fun/fun.rb
make test
```
didn't